config/initializers/rack_attack.rb

```diff
@@ -137,6 +137,11 @@ def path_without_extentions
     match_data[:account_id] if match_data.present?
   end
 
+  ## Prevent Abuse of attachment upload APIs ##
```

Comment:
❤️  

We can limit this to an account level to prevent abuse. similar to how we handle transcript endpoint

https://github.com/chatwoot/chatwoot/blob/2acf09b3eb1471d8a56ae5757aae88397a29287b/config/initializers/rack_attack.rb#L135

Reply:
Missed this comment, I'll handle this in a follow-up PR